tests/software/multiqc/main.nf

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { MULTIQC } from '../../../software/multiqc/main.nf' addParams(options: [publish_dir: 'test_multiqc'])
+
+
+workflow test_multiqc {
+
+    def input = []
+
```

Comment:
I think it would be better if it could create its own input data using the fastqc module, since the fastqc module should be relatively consistent. Please remove the input data then also.

Let me know if you need help with that!

Reply:
@Emiller88 , the tests are passing. 

But for this specific comment, I'm not sure what I need to do - could you please re-phrase? :)